src/Ucommerce.Masterclass.Umbraco/Api/MasterClassPaymentMethodController.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Web;
+using System.Web.Http;
+using MC_Headless.Exceptions;
+using Umbraco.Web.WebApi;
+
+namespace MC_Headless.Api
+{
+    public class MasterClassPaymentMethodController : UmbracoApiController
+    {
+        public MasterClassPaymentMethodController()
+        {
+        }
+
+        [HttpPost]
+        public IHttpActionResult Update()
+        {
+            var basketId = "";
+
+            if (string.IsNullOrWhiteSpace(basketId))
+                throw new MissingBasketIdException("Couldn't read basket id from cookies.");
+
+            var selectedPaymentMethodCookie = new HttpCookie("SelectedPaymentMethodId", Guid.NewGuid().ToString());
```

Comment:
Is the Guid.NewGuid() replaced in an exercise?

Reply:
Yes, that's the idea :) - I will put it in a variable instead and make it an empty string